### PR TITLE
update meeting notes page, +discourse badge on readme

### PR DESCRIPTION
**Pull request by jhamman**
cc @rabernat 

~Also, how do we add a link to the discourse site on the header bar?~

### CODE REVIEW

File: docs/meeting-notes.rst [hunk 1]
```diff
@@ -1,31 +1,51 @@
...
+----------------------
+
+1. Data Working Group
+    * Schedule: Every other Tuesday at 4p GMT
+    * Conferencing: `Zoom <https://zoom.us/j/5944845886>`_)
+    * Notes: `link <https://docs.google.com/document/d/e/2PACX-1vTP1iV4YJcHznwcfUwlLoEDVPdgtD9iAXql73vn4uAotOQMVh9R8vTIKxPHnvnrijX1C24T5aJJZ95x/pub>`_
+2. Machine Learning Working Group
+    * Schedule: First Monday of the month at 6p GMT
+    * Conferencing: Zoom (link TBD)
+    * Notes: (link TBD)
```
Comment (jhamman) [inline docs/meeting-notes.rst]: @nbren12 - can you help with these?

File: docs/meeting-notes.rst [hunk 2]
```diff
@@ -1,31 +1,51 @@
...
+3. Education Working Group 
+    * Schedule: (Schedule TBD)
+    * Conferencing: `whereby.com/pangeo <https://whereby.com/pangeo>`_
+    * Notes: (link TBD)
```
Comment (jhamman) [inline docs/meeting-notes.rst]: @robfatland - can you help with these?

File: docs/meeting-notes.rst [hunk 3]
```diff
@@ -1,31 +1,51 @@
...
+4. Cloud Operations Working Group
+    * Schedule: (Schedule TBD)
+    * Conferencing: `whereby.com/pangeo <https://whereby.com/pangeo>`_
+    * Notes: (link TBD)
```
Comment (jhamman) [inline docs/meeting-notes.rst]: Stay tuned here. I'm working on this one.

File: docs/meeting-notes.rst [hunk 4]
```diff
@@ -1,31 +1,51 @@
...
 
-.. _GCAL: https://calendar.google.com/calendar/embed?src=ucar.edu_c23ln4014khs3f65o93vqv5kqc%40group.calendar.google.com&ctz=America%2FLos_Angeles
-.. _ICAL: https://calendar.google.com/calendar/ical/ucar.edu_c23ln4014khs3f65o93vqv5kqc%40group.calendar.google.com/public/basic.ics
+Working Group Meetings
+----------------------
+
+1. Data Working Group
+    * Schedule: Every other Tuesday at 4p GMT
+    * Conferencing: `Zoom <https://zoom.us/j/5944845886>`_)
+    * Notes: `link <https://docs.google.com/document/d/e/2PACX-1vTP1iV4YJcHznwcfUwlLoEDVPdgtD9iAXql73vn4uAotOQMVh9R8vTIKxPHnvnrijX1C24T5aJJZ95x/pub>`_
```
Comment (jhamman) [inline docs/meeting-notes.rst]: @normanb - can you check over this?

Comment (normanb) [reply docs/meeting-notes.rst]: :+1:

**Comment by jhamman** [comment]
I've merged this PR. @nbren12, @robfatland - It would be great if you can follow up with a PR to update your respective meeting info blocks.

**Pull request merged**